src/PrestaShopBundle/Resources/config/services/adapter/shop.yml

```diff
@@ -48,3 +48,9 @@ services:
     tags:
       - name: 'tactician.handler'
         command: PrestaShop\PrestaShop\Core\Domain\Shop\Query\SearchShops
+
+  prestashop.adapter.shop.url.product_image_folder_provider:
+      class: 'PrestaShop\PrestaShop\Adapter\Shop\Url\ProductImageFolderProvider'
+      arguments:
+          - "@=service('prestashop.adapter.legacy.context').getContext().link"
+          - 'img/p'
```

Comment:
Should it be a parameter or a constant? This would allow us to make it different for `dev` and `test` environments

Reply:
I wanted to use the PS const but none is alright, `_PS_PROD_IMG_` doesn't seem to be accessible in cli or install environmnment and it didn't seem stable at all

So the hard coded value seems to be the safer choice for now, in the future I agree that this should be a parameter, but since this value is hard coded everywhere right now it doesn't make sense to ùake this configurable until we really refacto the whole image storage in the whole software